### DISCUSSION TITLE
Claude Sonnet 3.5 access via AWS Bedrock

### DISCUSSION THREAD

**Discussion by CrazyFunker** [zed-industries/zed#17636] [Feature Requests]
Hey all,

I would love to access the Anthropic Claude Sonnet 3.5 model via AWS Bedrock instead of using Anthropic APIs.
The use-case here is that in my company we have strict control over the data and security. As AWS partners we have access to Bedrock where the Sonnet 3.5 model is available and hosted securely.

Was that perhaps considered in the roadmap, would this be something worth pursuing by the developers of Zed?

Cheers

**Comment by CrazyFunker** [comment]
I see this has been raised already!
https://github.com/zed-industries/zed/issues/16544

**Comment by pietz** [comment]
AWS Bedrock Provider support would be great.

Many corporate users access their models through AWS because it's nicely integrated with all their other services.
